src/util/applyMixins.ts

```diff
@@ -0,0 +1,7 @@
+export default function applyMixins(derivedCtor: any, baseCtors: any[]) {
+  baseCtors.forEach(baseCtor => {
+    Object.getOwnPropertyNames(baseCtor.prototype).forEach(name => {
+      derivedCtor.prototype[name] = baseCtor.prototype[name];
+    })
+  });
+}
```

Comment:
Good point, in case someone uses applyMixins for anything beyond tests. Another alternative is to give an explicit name such as `verySlowApplyMixin`. I'll use `for` loops as you suggested.